### ISSUE TITLE
SimPLe efficient Agent?

### ISSUE THREAD

**Issue by jarlva** [tensorflow/agents#56]
Hi, any chance to port the SimPLe Agent from [https://github.com/tensorflow/tensor2tensor/tree/master/tensor2tensor/rl](url) to work with basic GYM (like cartpole)?

Based on the Google blog, SimPLe  "is significantly more efficient than current state-of-the-art techniques".

**Comment by sguada** [comment]
That would require some work, and probably not very relevant for a simple problem like cartpole

**Comment by jarlva** [comment]
Yes I know that cartpole is a simple one.. I picked cartpols as a simple example that we can learn to apply to more complicated ones. 
Again, based on Google's report this agent is unlike others. and will reduce training time more than any other agent to-date. Thus I think it's worth looking at. 
The red line is the figure is the Simple agent....

![Untitled](https://user-images.githubusercontent.com/26058035/55355020-d7fa9d80-54cf-11e9-811d-e59f35e6be58.png)

**Comment by sibyjackgrove** [comment]
Yes, adding SimPLe agent would be really beneficial. I am willing to help out if this is feasible with tf-agents.

**Comment by jarlva** [comment]
That would be awesome! (if it can be applied to GYM (non-Atari/ALE) environments.
Thanks.

**Comment by sguada** [comment]
It is feasible, but it would take a bit of work so before start working on it, you should create a proposal with detailed steps and milestones.
Also we should see how easy would be connecting TF-Agents with Tensor2Tensor, since you need that to train a good model.

**Comment by adityauser** [comment]
@sguada can you explain what do you mean by "connecting TF-Agents with Tensor2Tensor"?
Do you want Tensor2Tensor compatible in TF-Agents or implement SimPLe in TF-Agents from scratch or something else.

**Comment by jarlva** [comment]
Hello, checking to see if there is any progress?
